TransactionReporting/get-account-updater-job-details.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+var ApiContracts = require('authorizenet').APIContracts;
+var ApiControllers = require('authorizenet').APIControllers;
+var utils = require('../utils.js');
+var constants = require('../constants.js');
+var Map = require('collections/map');//added to iterate through loop by me
+
+function getaccountupdaterJobDetails(callback) {
+  var merchantAuthenticationType = new ApiContracts.MerchantAuthenticationType();
+	merchantAuthenticationType.setName(constants.apiLoginKey);
+	merchantAuthenticationType.setTransactionKey(constants.transactionKey);
+	var month = '2018-08';
+    var modifiedTypeFilter = 'all';
+    var refId='123456'
+    var paging = new ApiContracts.Paging();
+	paging.setLimit(36);
+	paging.setOffset(1);
+	var subscriptionId = [] ;
+    
+   var getRequest = new ApiContracts.GetAUJobDetailsRequest();
+	getRequest.setMerchantAuthentication(merchantAuthenticationType);
+	getRequest.setMonth(month);
+	getRequest.setPaging(paging);
+    getRequest.setModifiedTypeFilter(modifiedTypeFilter);
+    getRequest.setRefId(refId);
+    
+    var ctrl = new ApiControllers.GetAUJobDetailsController(getRequest.getJSON());
+      ctrl.execute(function () {
+		var apiResponse = ctrl.getResponse();
+		var response = new ApiContracts.GetAUJobDetailsResponse(apiResponse);
+		//pretty print response
+		//console.log(JSON.stringify(response, null, 2));
+		if (response != null) {
+			if (response.getMessages().getResultCode() == ApiContracts.MessageTypeEnum.OK) {
+				//console.log('Successfully created ' +  response.getAuDetails());
+				if (response.getAuDetails == null)
+					return response;
+				else{
+					console.log('refId :'+response.refId)
+					console.log('Result Code: ' + response.getMessages().getResultCode());
+					console.log('message:')
+				    console.log('Code: ' + response.getMessages().getMessage()[0].getCode());
+				    console.log('text: ' + response.getMessages().getMessage()[0].getText());
+				 //console.log('Successfully created audetails ' + JSON.parse(JSON.stringify(response.getAuDetails().getAuUpdateOrAuDelete())))
```

Comment:
There is no need to print these messages here unless an error message is present.